soh/src/code/z_eff_blure.c

```diff
@@ -199,100 +200,67 @@ void EffectBlure_Init2(void* thisx, void* initParamsx) {
 void EffectBlure_Destroy(void* thisx) {
 }
 
+void updateTrailColor(const char* trailName, Color_RGBA8* color, int* changed, int* reset) {
```

Comment:
I'm thinking these functions might be better moved to a c++ file. They're pretty nice to have and dont need to be specific to trails (I seem to recall seeing these patterns in other sections of the code). Small nitpick for this too would be the general naming format for functions in the codebase is generally pascal case, sometimes with an underscore as a separator for the object like family name and function purpose

Reply:
I looked for these patterns in the rest of the code and saw things like this:

`
   Color_RGB8 cButtonsColor = {255, 160, 0};
    if (CVarGetInteger("gCosmetics.Hud_CButtons.Changed", 0)) {
        cButtonsColor = CVarGetColor24("gCosmetics.Hud_CButtons.Value", cButtonsColor);
    }
`

Is it worth supporting RGBA and RGB in one function?

Reply:
I looked for these patterns in the rest of the code and saw things like this:

```
Color_RGB8 cButtonsColor = {255, 160, 0}; 
if (CVarGetInteger("gCosmetics.Hud_CButtons.Changed", 0)) { 
> cButtonsColor = CVarGetColor24("gCosmetics.Hud_CButtons.Value", cButtonsColor); 
} 
```

Is it worth supporting RGBA and RGB in one function?

Reply:
I looked for these patterns in the rest of the code and saw things like this:

```
Color_RGB8 cButtonsColor = {255, 160, 0}; if (CVarGetInteger("gCosmetics.Hud_CButtons.Changed", 0)) { 
> cButtonsColor = CVarGetColor24("gCosmetics.Hud_CButtons.Value", cButtonsColor);
}
```

Is it worth supporting RGBA and RGB in one function?